openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -128,7 +128,7 @@
 - control_key: SC-4
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: partial
```

Comment:
Does SELinux not cover this control? Also what happens, for example, to the memory of a pod after a pod is no longer in use? Does the memory still have access to old pieces of data after a new pod uses the memory blocks?